src/clj/harja/palvelin/raportointi/raportit.clj

```diff
@@ -212,24 +210,19 @@
     :urakkatyyppi #{:hoito}}
 
    {:nimi         :tyomaakokous
-    :parametrit   [{:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Materiaaliraportti"}
-                   {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Sanktioiden yhteenveto"}
+    :parametrit   [{:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Sanktioiden yhteenveto"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Muutos- ja lisätyöt"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Laskutusyhteenveto"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Laatupoikkeamat"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Soratietarkastukset"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Laaduntarkastusraportti"}
-                   {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Erilliskustannukset"}
-                   {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Yksikköhintaiset työt tehtävittäin"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Tiestötarkastukset"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Kelitarkastusraportti"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Ympäristöraportti"}
                    {:tyyppi "aikavali", :konteksti nil, :pakollinen true, :nimi "Aikaväli"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Ilmoitukset"}
                    {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Toimenpiteiden ajoittuminen"}
-                   {:tyyppi "checkbox", :konteksti "urakka", :pakollinen true, :nimi "Yksikköhintaiset työt päivittäin"}
```

Comment:
Oliko "Yksikköhintaiset työt päivittäin" tarkoituksella poistettu? En hoksannut tiketiltä eikä muokattuna excelissä.